CMakeLists.txt

```diff
@@ -87,15 +87,15 @@ configure_file(
 add_custom_target(uninstall
   "${CMAKE_COMMAND}" -P "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake")
 
-add_subdirectory("external")
 add_subdirectory("src")
 
 if(BUILD_TESTS)
   set(Pangolin_DIR ${Pangolin_BINARY_DIR}/src)
   add_subdirectory("test")
 endif()
 
-if(BUILD_EXAMPLES)
+if(BUILD_EXAMPLES AND NOT BUILD_EXTERN_GLEW)
```

Comment:
The user may have built GLEW elsewhere - GLEW is actually listed as a required dependency.